grails-app/views/login.gsp

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta name="layout" content="main"/>
+	<title>Log In - Groovy School</title>
+</head>
+<body class="full-screen login">
```

Comment:
the CSS classes in the body tag here will get overwritten by grails. reason why in the similar register.gsp i introduced the `<main class="full-screen register"> .... </main>` tag after the body